content/articles/introduction-to-subnetting/index.md

```diff
@@ -0,0 +1,205 @@
+
+Tabular representation of the differences between both versions of the internet protocol.
+
+| IPV4                    |     IPV6             |
+| ----------------------- | -------------------- |
+| The oldest version of the internet protocol | The newest version of the internet protocol |
+| 32 bits logical address  |  128 bits logical address  |
+| Uses dot-decimal notations, less suitable for mobile |  Uses hexadecimal colon-separated notations, best suited for mobile. |
+| Internet protocol security is optional  |  Internet protocol security is mandatory  |
+| The latest packet size is 576 bytes | The least packet size is 1208 bytes |
+| Its security depends on applications | Has it’s own security protocol called IPSec | 
+| Networks configured  with DHCP | Has its auto-configuration capabilities |
+| It is a numeric address method | It is an alphanumeric address method |
+| Has 5 different classes from A - E | Allows an unlimited number of IP addresses to be stored. |
+
+### Subnet Mask
+
+This is a term to denote the number of bits given to the network and the number given to the host. It is the process of setting the host bits to 0s and setting the network bits to 1s.  Here is the formula to calculate subnet;
+
+        For Network = 2n where n = number of desired bits. 
+        
+        For Hosts = 2n+2
+        
+        Example  to calculate subnets and hosts:
+        
+        Borrow 2 bits  (I.e n = 2)
+        
+| S   | S   | H   | H   | H   | H   | H   | H   |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+        Number(#) of subnets = 22 = 4. 
+        Subnet mask = 2bits = 128 + 64 = 192
+        Range of hosts = 26 = 64   
+
+|        | Range      | Usable Range |
+| ------ | ---------- | ------------ |
+| Network ID | 0-63 | 65 - 126 |
+|     |  64-127 | 129 - 190 |
+| Broadcast Address | 192 - 255 |     |
+
+To calculate subnets, networks, host ranges you will use a calculator. 
+[IPSubnet Mask Calculator for IPV4](https://www.site24x7.com/tools/ipv4-subnetcalculator.html) 
+
+### Routing
+
+This is the specialized process of connecting different IP networks. Hosts of different networks cannot communicate with each other until routed. 
+
+  ![routing](/introduction-to-subnetting/routing.jpg)
+
+  [Image source](https://computernetworkingnotes.com)
+
+IP routing process relies on two types of protocols; 
+Routed protocol and Routing protocol.
+
+**Routed Protocol:** This is a network protocol used to send user data between networks. It carries data such as web traffic, flies, messages, etc.
+
+**Routing Protocol**: A routed protocol finds paths (routes) for the IP and the routed protocol. Routing protocols run on Routers, Firewalls, Network servers with Network Operating Systems.
+
+Routing Protocols and the relating Routed Protocol
+| Routing Protocol   | Routed Protocol  |
+| ---------------- | ----------------- |
+| RIP, IGRP, OSPF, EIGRP | IP |
+| RIP, NLSP, EIGRP  | IPX  |
+| RMTP, AURP, EIGRP | Appletalk  |
+
```

Comment:
Consider adding a link to content where a curious reader can read further on these protocols